src/index.ts

```diff
@@ -1,10 +1,24 @@
+/* eslint-disable no-console */
 import { parse } from './command-parser/commandParser'
 import { CommandExecutor } from './commandExecutor'
+import { getErrorTranslation } from './errors'
+import NamedError from './namedError'
 import { CommandType } from './types'
 
-const parsedArgs = parse(process.argv)
-CommandExecutor().then(async (commandExecutor: any) => {
+const executeCommand = async (): Promise<void> => {
+  const { parser, parsedArgs } = parse(process.argv)
+  if (!Object.values(CommandType).includes(parsedArgs.command)) {
+    parser.print_help()
+    return
+  }
+  const commandExecutor = await CommandExecutor()
```

Comment:
why is CommandExecutor async? :thinking:

Reply:
Because it creates `cryptoProvider` inside, which is a async op.